src/main/kotlin/app/configuration/ContextConfiguration.kt

```diff
@@ -18,15 +18,15 @@ import java.security.SecureRandom
 class ContextConfiguration {
 
     @Bean
-    @Profile("production")
+    @Profile("strongRng")
```

Comment:
What's Rng? If it's obvious to others cool, but maybe a non-shortened word?

Reply:
Yeah we can do that. Random Number Generator is a bit long though?

Reply:
Fwiw I think rng here is fine. It's obvious to those familiar with crypto and now that Steven's been brought up to speed there's no need to change it imo.

Reply:
Makes sense to me 🙂